app/src/main/java/tech/pauly/findapet/discover/FilterBreedViewHolder.java

```diff
@@ -0,0 +1,31 @@
+package tech.pauly.findapet.discover;
+
+import android.support.v7.widget.RecyclerView;
+
+import tech.pauly.findapet.databinding.ItemFilterBreedBinding;
+import tech.pauly.findapet.databinding.ItemFiltersBinding;
+
+public class FilterBreedViewHolder extends RecyclerView.ViewHolder {
+
+    private ItemFiltersBinding itemFiltersBinding;
+    private ItemFilterBreedBinding itemFilterBreedBinding;
+
+    public FilterBreedViewHolder(ItemFiltersBinding binding) {
```

Comment:
A solution I've tried before that works well here, this really could be two types, but then you'd have to have a base class and everything...for something as small as two different types of views, one VH with different ways to bind works well. Yay data binding taking care of all of it!